spyder/plugins/pylint/main_widget.py

```diff
@@ -0,0 +1,899 @@
+            if self.error_output:
+                QMessageBox.critical(
+                    self,
+                    _("Error"),
+                    self.error_output,
+                )
+                print("pylint error:\n\n" + self.error_output, file=sys.stderr)
+            return
+
+        filename = self.get_filename()
+        rate, previous, results = self.parse_output(self.output)
+        self._save_history()
+        self.set_data(filename, (time.localtime(), rate, previous, results))
+        self.output = self.error_output + self.output
+        self.show_data(justanalyzed=True)
+        self.update_actions()
+        self.stop_spinner()
+
+    def _check_new_file(self):
+        fname = self.get_filename()
+        if fname != self.filename:
+            self.filename = fname
+            self.show_data()
+
+    def _is_running(self):
+        process = self._process
+        return process is not None and process.state() == QProcess.Running
+
+    def _kill_process(self):
+        self._process.kill()
+        self._process.waitForFinished()
+        self.stop_spinner()
+
+    def _update_combobox_history(self):
+        """Change the number of files listed in the history combobox."""
+        max_entries = self.get_option('max_entries')
+        if self.filecombo.count() > max_entries:
+            num_elements = self.filecombo.count()
+            diff = num_elements - max_entries
+            for __ in range(diff):
+                num_elements = self.filecombo.count()
+                self.filecombo.removeItem(num_elements - 1)
+            self.filecombo.selected()
+        else:
+            num_elements = self.filecombo.count()
+            diff = max_entries - num_elements
+            for i in range(num_elements, num_elements + diff):
+                if i >= len(self.curr_filenames):
+                    break
+                act_filename = self.curr_filenames[i]
+                self.filecombo.insertItem(i, act_filename)
+
+    def _save_history(self):
+        """Save the current history filenames."""
+        if self.parent:
+            list_save_files = []
+            for fname in self.curr_filenames:
+                if _('untitled') not in fname:
+                    list_save_files.append(fname)
+
+            self.curr_filenames = list_save_files[:MAX_HISTORY_ENTRIES]
+            self.set_option('history_filenames', self.curr_filenames)
+        else:
+            self.curr_filenames = []
+
+    # --- PluginMainWidget API
+    # ------------------------------------------------------------------------
+    def get_title(self):
+        return _("Code Analysis")
+
+    def get_focus_widget(self):
+        return self.treewidget
+
+    def setup(self, options):
+        change_history_depth_action = self.create_action(
+            PylintWidgetActions.ChangeHistory,
+            text=_("History..."),
+            tip=_("Set history maximum entries"),
+            icon=self.create_icon('history'),

```

Comment:
@goapeca seems like the mix of `'` and `"` is still there, maybe the rebase didn't pick the changes?

Reply:
You re right @dalthviz I pushed the rebase but not the commit for fixing the quotes. My bad.

Fixed noew.